scripts/issues/2102/ready-state.sql

```diff
@@ -3284,6 +3284,7 @@ COPY controlledvocabalternate (id, strvalue, controlledvocabularyvalue_id, datas
 22	United States of America	472	79
 23	U.S.A.	472	79
 24	YEMEN	483	79
+25  CSTR    825  30
```

Comment:
This file is historical/related to issue 2102. I don't think CSTR should be in this file in either section.

Reply:
I have Override it.